spec.html

```diff
@@ -6471,14 +6471,15 @@ <h1>
       <h1>
         CreateListFromArrayLike (
           _obj_: an ECMAScript language value,
-          optional _requirePropertyKeys_: a boolean,
+          optional _requirePropertyKeys_: a Boolean,
         ): either a normal completion containing a List of ECMAScript language values or a throw completion
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>It is used to create a List value whose elements are provided by the indexed properties of _obj_. _elementTypes_ contains the names of ECMAScript Language Types that are allowed for element values of the List that is created.</dd>
       </dl>
       <emu-alg>
+        1. If _requirePropertyKeys_ is not present, set _requirePropertyKeys_ to *false*.
```

Comment:
would it be simpler to change a few lines down, `If _requirePropertyKeys_ is present and _requirePropertyKeys_ is *true*,`?

Reply:
Yup, that'd be simpler.

(But note that simply prepending `_requirePropertyKeys_ is present and` to the condition doesn't quite work syntactically, because there's already 2 subconditions, so you need to change it to "If A, B, and C" syntax.)

Reply:
I'd rather do the defaulting up front, I think.